Dockerfile

```diff
@@ -46,4 +46,4 @@ RUN curl -sSL --output /tmp/nerdctl.tgz https://github.com/containerd/nerdctl/re
     tar zxvf /tmp/nerdctl.tgz -C /usr/local/bin/ && \
     rm -f /tmp/nerdctl.tgz
 
-FROM registry:2 AS registry2
+FROM registry:3.0.0-alpha.1 AS registry
```

Comment:
how long will this reference `registry:3.0.0-alpha.1` be valid? is this something we'll need to keep an eye on and replace with newer versions until the full release:3 is out, or can we be confident that this will work until then?